### DISCUSSION TITLE
Rejecting application for github students developer pack

### DISCUSSION THREAD

**Discussion by ghost** [github/india#453] [GitHub Platform]
Sir please please help me, I have try atleast seven times in defferent ways  , but they are rejecting my GitHub students developer pack application. Actually the problem they are showing that, there is not sufficient information for current enrollment date  and in my college I'd have only admission year with my photo or email id which used in college. Sir please help I don't what should I do next  . @Ey-mentor1 @EY-mentor2 @EY-mentor3  @ey-mentor4.

**Comment by FRTMentor** [comment]
Please fill all the data in you profile and try to resubmit.
And still facing any issues then please reach out the GitHub support team for more information.
